ReactNativeExample/app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "ReactNativeExample",
+  "displayName": "ReactNativeExample"
```

Comment:
Split React Native Example?

Reply:
:/ changing the app name requires additional changes in native code. I will leave it for a future iteration.